Vagrantfile

```diff
@@ -129,9 +129,11 @@ module Forklift
   plugin_vagrantfiles.each { |f| load f }
   plugin_base_boxes
   unsorted = @box_loader.add_boxes('boxes.yaml', 'config/versions.yaml') if File.exists?('boxes.yaml')
-  @boxes  = unsorted.keys.sort.inject({}) do |hash, key|
-    hash[key] = unsorted[key]
-    hash
+  if unsorted
+    @boxes  = unsorted.keys.sort.inject({}) do |hash, key|
```

Comment:
Right but `@boxes` contains everything, just as unsorted does, because add_boxes returns ALL boxes. @jlsherrill was there a reason you couldn't just store the result in `@boxes` and then sort that?